src/main/java/com/company/elixr/springbootapplicationwithdocker/model/FileAccessByUserNameDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.company.elixr.springbootapplicationwithdocker.model;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Component
+@Getter
+@Setter
+@Builder
+public class FileAccessByUserNameDTO {
```

Comment:
Not a fan of the name. Too complex and does not really make the intention clear

Reply:
Changed the name to FileInfoDTO